python/benchmark/dataproc/README.md

```diff
@@ -0,0 +1,62 @@
+# Benchmarking on Dataproc
+
+This directory contains shell scripts for running larger scale benchmarks on a Google Dataproc cluster.  You will need a Dataproc account to run them.  The benchmarks use datasets synthetically generated using [gen_data.py](../gen_data.py).  For convenience, these have been precomputed and currently stored in the public GCS bucket `gs://spark-rapids-ml-benchmarking/datasets`.  The benchmark scripts are currently configured to read the data from there.
```

Comment:
The gs bucket is not public.    Let's not make it public for now.   Somehow reword this comment.

Reply:
Updated.